rpc/backend/utils.go

```diff
@@ -257,20 +257,17 @@ func TxLogsFromEvents(events []abci.Event, msgIndex int) ([]*ethtypes.Log, error
 
 // ParseTxLogsFromEvent parse tx logs from one event
 func ParseTxLogsFromEvent(event abci.Event) ([]*ethtypes.Log, error) {
-	logs := make([]*evmtypes.Log, 0, len(event.Attributes))
```

Comment:
`logs := make([]*evmtypes.Log, 0, len(event.Attributes))` preallocates memory. Curious, why it was removed?